executor/prepared.go

```diff
@@ -235,12 +241,16 @@ func (e *ExecuteExec) Build() error {
 		return errors.Trace(ErrWrongParamCount)
 	}
 
+	if cap(vars.PreparedParams) < len(e.UsingVars) {
+		vars.PreparedParams = vars.PreparedParams[:len(e.UsingVars)]
```

Comment:
Do not expand slice `vars.PreparedParams` like this.

When `len(e.UsingVars) > len(vars.PreparedParams)`, which is definitely in the `if` statement, this will cause a panic like: `runtime error: slice bounds out of range`

Reply:
Thank you for the comment. I'v corrected the codes as follows:

`   if cap(vars.PreparedParams) < len(e.UsingVars) {`
`-      vars.PreparedParams = vars.PreparedParams[:len(e.UsingVars)]`
`+      vars.PreparedParams = make([]interface{}, len(e.UsingVars))`
`   }`